lib/gooddata/lcm/actions/collect_ldm_objects.rb

```diff
@@ -28,7 +28,10 @@ def call(params)
           synchronize = params.synchronize.pmap do |info|
             from = info.from
             from_project = development_client.projects(from) || fail("Invalid 'from' project specified - '#{from}'")
-            objects = (from_project.attributes.to_a + from_project.labels.to_a + from_project.datasets.to_a + from_project.facts.to_a).map(&:uri)
+            # TMA-836 - reading objects from datasets includes deprecated ones
+            attributes = from_project.datasets.map(&:attribute_uris).flatten
```

Comment:
Refers to 'from_project' more than self (maybe move it to another class?) (FeatureEnvy)